qiskit_experiments/calibration/calibrations.py

```diff
@@ -897,6 +899,78 @@ def parameters_table(
 
         return data
 
+    def update(
+        self,
+        exp_data: ExperimentData,
+        result_index: int = -1,
+        force_update: bool = False,
+        group: str = "default",
+    ):
+        """Update the calibrations form a result in the given experiment data.
+
+        This function allows users to update their calibrations from experiment data. Typically,
+        the value of the parameter to update is directly stored as the result of the fit. However,
+        for more complex cases, such as Rabi, the value of the parameter is extracted from the
+        fit result using the calibration extraction callable. Importantly, this method requires
```

Comment:
Is this still true? I think the implementation of Rabi parameter extraction should be discussed in separate PR. I think directly updating analysis of Rabi experiment should be fine since this is quite rare case.

Reply:
Thanks. I updated it here: 722ba59